bcda/client/bluebutton.go

```diff
@@ -89,7 +89,7 @@ func NewBlueButtonClient(config BlueButtonConfig) (*BlueButtonClient, error) {
 		return nil, errors.Wrap(err, "could not load Blue Button keypair")
 	}
 
-	tlsConfig := &tls.Config{Certificates: []tls.Certificate{cert}, MinVersion: tls.VersionTLS12}
+	tlsConfig := &tls.Config{Certificates: []tls.Certificate{cert}, MinVersion: tls.VersionTLS12, NameToCertificate: nil}
```

Comment:
This was due to a deprecation notice in the build pipeline. May you please verify this is correct?